SampleNHAppSwiftUI2/SampleNHAppSwiftUI2/ContentView.swift

```diff
@@ -0,0 +1,68 @@
+//
```

Comment:
Please remove all of your personal headers and use the Microsoft header for the rest of the samples